src/robot/libraries/dialogs_ipy.py

```diff
@@ -12,32 +12,183 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import wpf
```

Comment:
Turns out that ```import wpf``` is pulling in some required assemblies behind the scenes. It can be done in other ways, but they are all uglier and much more verbose. I'll add some kind of comment.